src/lib/auth/linking.ts

```diff
@@ -1,14 +1,28 @@
 import * as webnative from 'webnative'
-import type { account } from 'webnative'
+import { get as getStore } from 'svelte/store'
+
+import { programStore } from '$src/stores'
+
 
 export const createAccountLinkingConsumer = async (
   username: string
-): Promise<account.AccountLinkingConsumer> => {
-  return await webnative.account.createConsumer({ username })
+): Promise<webnative.AccountLinkingConsumer> => {
+  const program = getStore(programStore)
+  if (program) return program.auth[ webnative.strategyTypes.default ].accountConsumer(username)
+
+  // Wait for program to be initialised
+  return new Promise((resolve) => {
+    programStore.subscribe(program => {
+      if (!program) return
+      const consumer = program.auth[ webnative.strategyTypes.default ].accountConsumer(username)
```

Comment:
Mentioning this here as a follow-up from our meeting that we'll want to add `program.auth[ webnative.strategyTypes.default ]` to `sessionStore.program`. Maybe i'll actually PR that change against this branch :)